tests/unit/h/accounts/schemas_test.py

```diff
@@ -81,26 +82,21 @@ def test_it_is_invalid_when_password_too_short(self, pyramid_request):
             schema.deserialize({"password": "a"})
         assert exc.value.asdict()["password"] == ("Must be 8 characters or more.")
 
-    def test_it_is_invalid_when_username_too_short(self, pyramid_request, user_model):
+    def test_it_is_invalid_when_username_too_short(self, pyramid_request):
         schema = schemas.RegisterSchema().bind(request=pyramid_request)
-        user_model.get_by_username.return_value = None
```

Comment:
In most cases the mocking was actually unnecessary: the real `get_by_username()` will return `None` anyway.